src/modules/oAuth/oAuth.module.ts

```diff
@@ -0,0 +1,13 @@
+import { GoogleStrategy } from './strategies/google.strategy';
+import { Module } from '@nestjs/common';
+import { AppController } from './oAuth.controller';
+import { OAuthService } from './oAuth.service';
+import { ConfigModule } from '@nestjs/config';
+import { UsersModule } from '../users/users.module';
+
+@Module({
+  imports: [ConfigModule.forRoot(), UsersModule],
```

Comment:
I think ConfigModule can be omitted because it is imported globally in app.module.ts. You can directly use it with injection.

Reply:
I needed to change on app.module to remove it